beacon-chain/state/v1/unsupported_getters.go

```diff
@@ -34,3 +34,13 @@ func (*BeaconState) NextSyncCommittee() (*ethpb.SyncCommittee, error) {
 func (*BeaconState) LatestExecutionPayloadHeader() (*ethpb.ExecutionPayloadHeader, error) {
 	return nil, errors.New("LatestExecutionPayloadHeader is not supported for phase 0 beacon state")
 }
+
+// ProveFinalizedRoot --
+func (b *BeaconState) ProveFinalizedRoot() ([][]byte, error) {
```

Comment:
This should work for v1, but not the sync committee proofs, so just laying out this pattern once that is added